packages/assets-controllers/src/TokensController.ts

```diff
@@ -570,12 +570,24 @@ export class TokensController extends BaseController<
    * Ignore a batch of tokens.
    *
    * @param tokenAddressesToIgnore - Array of token addresses to ignore.
+   * @param networkClientId - Optional network client ID used to determine interacting chain ID.
    */
-  ignoreTokens(tokenAddressesToIgnore: string[]) {
+  ignoreTokens(
+    tokenAddressesToIgnore: string[],
+    networkClientId?: NetworkClientId,
```

Comment:
In the future we can make more of things chain ids instead of network client ids.  so we don't have to convert between them.  Would also simplify the client code.  But I'll approve this for now since you have it working.